Components/BarChart/Menu/index.js

```diff
@@ -0,0 +1,49 @@
+import styled from "styled-components";
+
+const StyledSelect = styled.select`
+  margin: 0;
+  padding: 0;
+  background-color: #cee0ed;
+  border-radius: 3px;
+  border: 1px soild black;
+  margin-bottom: 0.5rem;
+`;
+
+export default function BarChartSelect({ setBarSelection, setLabel }) {
+  const handleSelectChange = (event) => {
+    setBarSelection(event.target.value);
+    setLabel(event.target.value);
```

Comment:
why are these two different states? If they always hold the same value, please only use one state. :)

Reply:
I wanted to add another logic to the label initially, to have it render a descriptive string rather than the props name (like fg_pct). But I discarded that thought for now and thus removed the redundant state ;)

Reply:
✅